external/protobuf/build_protobuf.cmake

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+cmake_minimum_required(VERSION 3.15)
+
+# Trickery to circumvent https://gitlab.kitware.com/cmake/cmake/-/issues/19703
+# Avoids rebuilding if the git was in the same state as in the previous build.
+
+execute_process(
+  COMMAND /usr/bin/git log -n 1 --pretty=format:%H
```

Comment:
Hardcoded git path here seems a bit brittle. Not sure if it's worth fixing?

Reply:
Either way, this means `git` package needs to be added to `INSTALL.md` command for cmake build deps

Reply:
Added git to the instructions and fixed command to use `GIT_EXECUTABLE` variable.

Reply:
Thanks!